roles/ironicclient/tasks/_suse.yml

```diff
@@ -4,7 +4,7 @@
     name: vm
     tasks_from: install_suse_package
   vars:
-    suse_packages: [python310, python310-pip, python310-devel, gcc]
+    suse_packages: [python310, python310-pip, python310-devel, gcc, python310-dbm]
```

Comment:
Please let me know if I'm wrong but I believe this should be sufficient for the core VM. Do we have to also list the package in the `capi` and `kubernetes-tools` roles?